### PR TITLE
Updated error handling and logging

### PR DESCRIPTION
**Pull request by ellynch-r7**

### CODE REVIEW

**Comment by stopal-r7** [comment]
This could be a part of https://github.com/rapid7/lecli/pull/32 PR

Updating version number with the related code change is better practice imho.

File: lecli/team_api.py [hunk 1]
```diff
@@ -92,13 +93,13 @@ def create_team(name):
         response = requests.post(_url(), json=params, headers=headers)
         if response_utils.response_error(response):
             print 'Creating team failed, status code: %d' % response.status_code
```
Comment (udangel-r7) [inline lecli/team_api.py:95]: if we are adjusting things, this error message probably should also go to stderr

Comment (ellynch-r7) [reply lecli/team_api.py:95]: Sure, I'll update this and have a look to see if there's any more instances of this

**Comment by ellynch-r7** [comment]
Updated PR, @dtracey-r7  @udangel-r7 can you have another look please?

File: lecli/response_utils.py [hunk 2]
```diff
@@ -10,22 +11,22 @@ def response_error(response):
     """
     if response.headers.get('X-RateLimit-Remaining') is not None:
         if int(response.headers['X-RateLimit-Remaining']) == 0:
-            print 'Error: Rate Limit Reached, will reset in ' + response.headers.get(
-                'X-RateLimit-Reset') + ' seconds'
+            sys.stderr.write('Error: Rate Limit Reached, will reset in ' + response.headers.get(
+                'X-RateLimit-Reset') + ' seconds')
```
Comment (udangel-r7) [inline lecli/response_utils.py]: you probably need a newline here?

File: lecli/response_utils.py [hunk 3]
```diff
@@ -10,22 +11,22 @@ def response_error(response):
...
             return True
     try:
         response.raise_for_status()
     except requests.exceptions.HTTPError as error:
         print "Request Error:", error.message
         if response.status_code == 500:
-            print 'Your account may have no owner assigned. ' \
-                  'Please visit www.logentries.com for information on assigning an account owner.'
+            sys.stderr.write('Your account may have no owner assigned. ' \
+                  'Please visit www.logentries.com for information on assigning an account owner.')
```
Comment (udangel-r7) [inline lecli/response_utils.py]: newline at the end?

**Review by udangel-r7 — APPROVED** [review]

**Comment by dtracey-r7** [comment]
+1

**Pull request merged**